web-components/src/components/tooltip/scss/tooltip.scss

```diff
@@ -32,7 +32,7 @@
       text-align: center;
       width: max-content;
       word-break: break-word;
-      z-index: -1;
+      z-index: 9999;
```

Comment:
Lena you can't simply put 9999. Please evaluate other z-indexs. there is a hierarchy here: https://sqbu-github.cisco.com/CBABU/agentx-services/blob/master/src/styles/z-index.scss

Reply:
Fixed